docs/reference/api/connectivity/lorawan/lorawan.md

```diff
@@ -0,0 +1,70 @@
+<h2 id="lorawan-api">LoRaWAN</h2>
+
+LoRaWAN is a technology designed for low-power battery powered devices. These devices operate in an unlicensed spectrum, creating high desnity wide-area networks.
+
+Arm Mbed OS provides a native network stack for LoRaWAN, which can run on any Mbed Enabled device with a LoRa radio onboard. 
+
+The [LoRaWANInterface](https://github.com/ARMmbed/mbed-os/blob/feature-lorawan/features/lorawan/LoRaWANInterface.h) provides a C++ API for connecting to the internet over a LoRa network.
+
+## LoRaWANInterface class reference
+
+[![View code](https://www.mbed.com/embed/?type=library)](http://os-doc-builder.test.mbed.com/docs/v5.8/mbed-os-api-doxy/class_l_o_r_a_w_a_n_interfcae.html)
+
+## Usage
+
+To bring up the Mbed LoRaWAN stack, consider the following progression: 
+
+1) An [EventQueue](https://os.mbed.com/docs/v5.8/reference/eventqueue.html) object. 
```

Comment:
Query: Do these numbers indicate steps that need to happen in order? Or is it more of a logical grouping of items? [This only matters because we use numbers for ordered steps and bullets for unordered groupings.]

Reply:
Its an ordered progression.

Reply:
This is valid link
https://os-doc-builder.test.mbed.com/docs/development/mbed-os-api-doxy/classevents_1_1_event_queue.html